R/utils-format.r

```diff
@@ -277,6 +277,21 @@ tibble_glimpse_width <- function(width) {
   getOption("width")
 }
 
+pluralise_msg <- function(message, objects) {
+  paste0(pluralise(message, objects), format_n(objects))
+}
+
+pluralise <- function(message, objects) {
+  stopifnot(length(objects) > 0)
+  if (length(objects) == 1) {
+    # strip [, remove everything within ()
+    gsub("\\[|\\]|\\([^\\)]+\\)", "", message)
```

Comment:
I'm not sure I follow the examples or understand the use case, but I did follow your instruction. Now if pairs of parens / brackets have spaces inside, the regex leaves them and their contents alone.